src/parsers/chrono/chrono.js

```diff
@@ -56,6 +56,31 @@ dayPeriodsParser.extract = (text, ref, match) => {
   });
 };
 
+const currentMonthDayParser = Object.assign(
+  new chrono.Parser(),
+  {
+    pattern: () => /\b(?:on|by)\s+the\s+(\d\d?)(?:th|st|rd|nd)\b/,
```

Comment:
* For the sake of consistency, let's reuse what is in the original parsers. See https://github.com/wanasit/chrono/blob/master/src/parsers/EN/ENMonthNameMiddleEndianParser.js#L30 for how the days regexp is usually formatted:
```js
([0-9]{1,2})(?:st|nd|rd|th)?
```
* Also, note that the suffix is made optional which I think may sense so that 'the 12' is parsed as well.
* Then, all inputs are normalised, so you can safely replace `\s+` by `\s` or even better by just ` ` for greater readibility.
* Finally I wonder if we'll ever get fully written numbers like 'first' or 'second'. That, of course, depends on the output of our speech recogniser. If this is the case, we should probably add a numeric normaliser during the cleaning phase to handle this. The library does through https://github.com/wanasit/chrono/blob/master/src/utils/EN.js#L74-L106 and we can probably import that object as is.